1-exercises/E-for-loop/exercise2.js

```diff
@@ -26,7 +26,11 @@ const AGES = [
     49
 ];
 
-// TODO - Write for loop code here
+WRITERS.forEach(WRITERS); {
+let sentence = "(WRITERS.name)is (AGES.num) years old"    
+console.log(sentence)
+};
+//GR - again, not sure about the error message I'm getting here - will come back to it//
```

Comment:
when i run this , it says writers is not defined

Reply:
have a look here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for